common/src/main/java/xyz/wagyourtail/jsmacros/client/api/helpers/FullOptionsHelper.java

```diff
@@ -0,0 +1,2374 @@
+package xyz.wagyourtail.jsmacros.client.api.helpers;
+
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.option.AoMode;
+import net.minecraft.client.option.AttackIndicator;
+import net.minecraft.client.option.ChatVisibility;
+import net.minecraft.client.option.CloudRenderMode;
+import net.minecraft.client.option.GameOptions;
+import net.minecraft.client.option.GraphicsMode;
+import net.minecraft.client.option.KeyBinding;
+import net.minecraft.client.option.NarratorMode;
+import net.minecraft.client.option.ParticlesMode;
+import net.minecraft.client.option.SimpleOption;
+import net.minecraft.client.render.ChunkBuilderMode;
+import net.minecraft.client.render.entity.PlayerModelPart;
+import net.minecraft.client.resource.language.LanguageDefinition;
+import net.minecraft.client.sound.SoundManager;
+import net.minecraft.network.packet.c2s.play.UpdateDifficultyLockC2SPacket;
+import net.minecraft.resource.ResourcePackManager;
+import net.minecraft.resource.ResourcePackProfile;
+import net.minecraft.sound.SoundCategory;
+import net.minecraft.text.Text;
+import net.minecraft.text.TranslatableTextContent;
+import net.minecraft.util.Arm;
+import net.minecraft.world.Difficulty;
+
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.lang3.ArrayUtils;
+import xyz.wagyourtail.jsmacros.client.mixins.access.MixinSimpleOption;
+import xyz.wagyourtail.jsmacros.core.helpers.BaseHelper;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Locale;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+/**
+ * @author Etheradon
+ * @since 1.8.4
+ */
+@SuppressWarnings("unused")
+public class FullOptionsHelper extends BaseHelper<GameOptions> {
```

Comment:
While we are at it, I suggest to not strip the deprecated javadoc text from the website, because it can be quite confusing to figure out why something is deprecated or what to use instead.